_posts/2020-03-25-top-reinvent-videos.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: post
+title: Re:Invent Yourself and Your Company With Serverless
+date: 2020-03-25
+tags: serverless aws security reinvent
+author: Matthew Tyler
+image: img/reinvent.png
+---
+
+<center><img src="/img/reinvent.png" /></center>
+<br/>
+
+# Introduction
+
+I became a father for the first time recently. Turns out your hands are pretty much full for a lot of the time, rocking an upset infant back-and-forth in an attempt to get them to settle. Watching all the re:invent content I wasn't able to attend when I was at the conference seemed like a good way to fill in the time. I thought I'd put together an expansive list of my favourite videos from the last couple two re:invents. These sessions run a bit of a gamut; some are what I consider base-line knowledge, some are advanced use-cases, some are just interesting. Generally, they focus around the serverless ecosystem and topics that I think developers needs to know to become expert serverless practictioners.
```

Comment:
Great intro.

Baseline vs expert - can you add something to each heading?  Level or such - so people can skim past easily?

Reply:
gives a hook for people at all levels to choose their own adventure - rather than TL;DR

Reply:
All the videos have number codes, I'll add something to explain it.